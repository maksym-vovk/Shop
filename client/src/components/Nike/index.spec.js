import React from 'react';
import { Nike } from './index';

describe('Nike component', () => {
  it('Nike component render', () => {
    const wrapper = shallow(<Nike />);
    expect(wrapper).toMatchSnapshot();
  });
});