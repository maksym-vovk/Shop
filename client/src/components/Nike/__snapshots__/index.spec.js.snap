// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nike component Nike component render 1`] = `ShallowWrapper {}`;
